db/migrate/20230921021410_add_tweet_to_bookmarks.rb

```diff
@@ -0,0 +1,5 @@
+class AddTweetToBookmarks < ActiveRecord::Migration[7.0]
+  def change
+    add_reference :bookmarks, :tweet, null: false, foreign_key: true
```

Comment:
same here bookmarks already has the tweet_id column, this result in `PG::DuplicateColumn: ERROR: `